libcaf_test/caf/test/runner.cpp

```diff
@@ -68,7 +67,15 @@ int runner::run(int argc, char** argv) {
     return EXIT_SUCCESS;
   }
   default_reporter->start();
+  auto enabled = [](const std::regex& selected,
+                    const std::string_view& search_string) {
+    return std::regex_search(search_string.begin(), search_string.end(),
+                             selected);
+  };
+  auto suite_regex = parse_regex_opt(get_or(cfg_, "suites", ".*"));
```

Comment:
With this implementation, we silently drop errors if users pass in invalid options. We should stop the application here instead and only carry on if the regex is valid:

```suggestion
  auto suite_regex = parse_regex_opt(get_or(cfg_, "suites", ".*"));
  if (!suite_regex) {
    // print error and return
  }
```